config/webpack.config.js

```diff
@@ -52,7 +52,8 @@ let cfg = {
   },
 
   resolve: {
-    extensions: ['.ts', '.tsx', '.js', '.json'],
+    fullySpecified: false,
```

Comment:
@kr8n3r I've had some time to dig into this and I think `fullySpecified` needs to be defined in a resolve block within the `module: { rules: [] }` section below, like so:

```
module: {
    rules: [
      {
        test: /\.mjs$/,
        type: 'javascript/auto',
        resolve: {
          fullySpecified: false
        }
      }
    ]
}

If you do this, I think you can remove the hardcoded file extensions in the govuk-esm javascript files.

https://webpack.js.org/configuration/module/#resolvefullyspecified